great_expectations/data_context/data_context_variables.py

```diff
@@ -106,12 +106,12 @@ def _get(self, attr: DataContextVariableSchema) -> Any:
         substituted_val: Any = substitute_all_config_variables(val, self.substitutions)
         return substituted_val
 
-    def save_config(self) -> None:
+    def save_config(self) -> Any:
         """
         Persist any changes made to variables utilizing the configured Store.
         """
         key: ConfigurationIdentifier = self.get_key()
-        self.store.set(key=key, value=self.config)
+        return self.store.set(key=key, value=self.config)
```

Comment:
Small change so we can assert that `self.store.set` returns `True` in a cloud environment - useful to check that the endpoint is functional.